sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,174 @@
+- annotated_ptr API fixes [954730e7a8232087f9bca19b26a7c2f8d5bf08f3] [fbad42a398efca3aa04bdba6504885abc9b333f7] [43336a15fcafec1b3c3e94753023304ec6ef0ed3]
+- Implement fallback path for profiling info support [7cfa951303eb828836637c21518ed0f1d27ad753]
+- Fix require alignment in buffers [093dae1f72146ef0f640d2e667576811245eb078]
+- Throw an exception when unsupported features used in a graph [8d8d3f4f52b0c7710bafdb6acd7d9157c6a0b7d4]
+- Support get_multi_ptr in ESIMD context [2f0f3814c5415068ad77ae954580b4a7618cccdd]
+- Accept integral type in parallel_for with offsets [ecfc49c825c665047ada8be42ccb3202e4111227]
+- Support assert() in ESIMD kernels [545a043ebaf52976b4a3c7bf35e2ea637428c505]
+- Add missing Intel archs to sycl_ext_oneapi_device_architecture [c4db251179dcd1b30a1043d4780634dbe39b3e3a]
+- Enable range rounding for unnamed lambdas [4f30e6648d9e38fefcc1f606638ea6d6604390bb]
+- Replaced LoadLibraryA() with LoadLibraryEx() [713442327d4038d5cf2158c1d3d85920a30d61cf]
+- Don't use legacy ANSI-only Windows API for loading plugins [5c30815d48211565812adb68f95d9b6fb788a39e]
+- Use SFINAE to reject wrong signature of CGF object parameter [5801970585b2724dae8367470152304cd2718e3c]
+- fill() use native functions instead of custom kernels [0249a55f810eedc42c3571cff176c5ab01606793]
+- Add defaults for streaming and register_map fpga_kernel_properties [23a6f389c1e45df077c6f15b691835b2976fda4d]
+- Move GRF property conflict check to compile time [4fb92d35c7beac85913edd23197c2e6506e49240]
+- Add error when using invoke_simd with callables with struct arguments [03c13fd53a2413de41b68fd03d7ad7e7ef905215]
+- Add missing rintf [3c327c736ced3690b283f04302146ac952f53726]
+- Cache the max local mem size in CUDA adapter [1c892f952cae7f0cea5c446534441478948e3f7d]
+- Add support for local accessors to lsc API [9dce2d227b31fdbba09a044500a0ea849cdeb747]
+- Replace USM allocator with UMF abstractions in Level Zero backend [7b9b8ce4e4c5be5503b22e4cedf341b4120d1ad6]
+- Make SYCL-Graph functions thread-safe [c8c64a673eebfede0d9dbc0728a4eaaa5bde0b10]
+- Improve diagnostic for nd_item passed to parallel_for with range [e600d7922489635290fd2cb63300d865fc72ee72]
+- Implement missing exceptions defined by SYCL-Graphs specification [77b794bcaad6a33ff5734d938412f61b6b7475f0]
+- Implemented supported make_device, make_queue, make_event interop functions for HIP backend [5e9d07b14a47c2097764b148c3b1dcdb9fcbbc1c]
+- Add missing nearbyint for CUDA [0ef26d3ee8086cb51d4f5deb9ad8c3d81cdd8f16]
+- Implement runtime query for matrix combinations [a09491f4e389dbef29128bda69a7954e4b887af6]
+- Support basic kernel launches over 64-bit global ranges [192c3016e76b36b755bd13b1d32446071812c600]
+- Clarify runtime error for nested kernel submissions [7afc2d03fd0040177009967aca057cc80708747b]
+- Improve handling of spec constants with padding in the middle of structs [292309c826a3e11b8cb225667b316d8ce01e2779]
```

Comment:
We can also probably say "fix compiler crash when spec constant is an unpacked struct with padding in the middle" - that is probably more descriptive for an end user than just "improve handling"